src/components/headerImageCaption.tsx

```diff
@@ -75,18 +62,28 @@ const HeaderImageCaptionStyles = css`
 	}
 `;
 
+const svgStyle = css`
+	line-height: 55px;
```

Comment:
Can we convert this to `rem` for scaling purposes? I think it will be approximately `3.44rem`.